models/DataObject/ClassDefinition/Data/AdvancedManyToManyRelation.php

```diff
@@ -205,6 +205,7 @@ public function getDataForQueryResource($data, $object = null, $params = [])
         $d = [];
 
         if (is_array($data) && count($data) > 0) {
+            /** @var DataObject\Data\ElementMetadata $metaObject */
```

Comment:
Than we can set the $data param to `DataObject\Data\ElementMetadata[]|null`, right?

Reply:
isn't this already the case?

Reply:
No it's `Element\ElementInterface[]|null`. This var comment should no longer be necessary after that.

Reply:
Ah, now I got you! thx